COPYRIGHT

```diff
@@ -17,3 +17,7 @@ operating system images.
 macros/rust and macros/cargo (used during build) are derived from the Fedora Rust SIG's rust2rpm.
 https://pagure.io/fedora-rust/rust2rpm
 Copyright (c) 2017 Igor Gnatenko
```

Comment:
Can you add a separate commit that removes this? It's no longer applicable here; we need to add it to COPYRIGHT in the sdk repo.